helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -583,8 +580,7 @@ private void subscribeForChanges(NotificationContext.Type callbackType, String p
    * exists
    */
   public void init() {
-    logger.info("initializing CallbackHandler: " + this.toString() + " content: " + getContent());
-
+    logger.info("Initializing CallbackHandler, content: {}", getContent());
```

Comment:
Why reduce the information here? The logger is constructed with CallbackHandler class, it is not clear for us which child class it might be if you remove the this.toString().

Reply:
The `getContent()` method has the richest information to tell everything
```
public String getContent() {
    return "CallbackHandler{" + "_watchChild=" + _watchChild + ", _preFetchEnabled="
        + _preFetchEnabled + ", _batchModeEnabled=" + _batchModeEnabled + ", _path='" + _path + '\''
        + ", _listener=" + _listener + ", _changeType=" + _changeType + ", _manager=" + _manager
        + ", _zkClient=" + _zkClient + '}';
  }
```